From 1290efb3b663169666fd9428d3b1fd6c03f984ba Mon Sep 17 00:00:00 2001
From: Klas Lindfors <klas@yubico.com>
Date: Mon, 20 Apr 2015 10:53:08 +0200
Subject: [PATCH] add no_testcase_installs for tests to avoid installation

fixes #49
---
 tests/tests.pro | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/tests.pro b/tests/tests.pro
index a49ca54..8a11661 100644
--- a/tests/tests.pro
+++ b/tests/tests.pro
@@ -2,7 +2,7 @@ QT += testlib
 QT -= gui
 SOURCES += test.cpp scanedittest.cpp yubikeyutiltest.cpp
 HEADERS += scanedittest.h yubikeyutiltest.h
-CONFIG += testcase
+CONFIG += testcase no_testcase_installs
 CONFIG += console
 CONFIG -= app_bundle
 INCLUDEPATH += ../lib
-- 
2.1.4

